eva/executor/create_udf_executor.py

```diff
@@ -46,6 +48,12 @@ def handle_huggingface_udf(self):
             self.node.metadata,
         )
 
+    def handle_ultralytics_udf(self):
+        """Handle Ultralytics UDFs"""
+        # manually set the impl_path for yolo udfs
+        self.node._impl_path = Path(f"{EVA_DEFAULT_DIR}/udfs/yolo_object_detector.py")
```

Comment:
Adding decorators to this file should fix the decorators problem. Because the create_udf_executor will read decorators from here and update it in the catalog.